src/lib/compile-route.js

```diff
@@ -16,13 +16,25 @@ function normalizeRequest (url, options, Request) {
 		};
 	} else {
 		return {
-			url: url,
+			url: normalizeURL(url),
 			method: options && options.method || 'GET',
 			headers: options && options.headers
 		};
 	}
 }
 
+function normalizeURL (url) {
+	var r = new RegExp('^(?:[a-z]+:)?//', 'i'); // https://stackoverflow.com/a/19709846/308237
```

Comment:
could you move this comment above the line and add some text explaining it is testing for the part of the url before path